app/routes/spending.py

```
@@ -0,0 +1,114 @@
+            return []
+        query = model.query.filter(model.vehicle_id.in_(vehicle_ids))
+        if vehicle_filter:
+            query = query.filter(model.vehicle_id == vehicle_filter)
+        return query.all()
+
+    fuel_logs = fetch(FuelLog) if fuel_enabled else []
+    charging = fetch(ChargingSession) if fuel_enabled else []
+    expenses = fetch(Expense) if expenses_enabled else []
+
+    totals = {
+        'fuel': (sum(l.total_cost or 0 for l in fuel_logs)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)